lib/ex_web3_ec_recover/signature.ex

```diff
@@ -14,10 +14,12 @@ defmodule ExWeb3EcRecover.Signature do
       binary_part(sig_binary, 64, 1)
       |> :binary.decode_unsigned()
 
+    v_num = if v_num >= 27, do: v_num - 27, else: v_num
```

Comment:
https://eips.ethereum.org/EIPS/eip-155

Reply:
This value is suppose to be either 0 or 1, based on parity of some other stuff used for calculation or `{0,1} + 27`. I wonder whether we should only support these 4 values for security reasons.

Reply:
We totally can put that in although I believe the EthereuemJS library (method fromRpcSig in signature.ts)  is doing pretty much the same thing as what is added here.  

https://github.com/ethereumjs/ethereumjs-util/blob/master/src/signature.ts (Line 112)
```
 let v = bufferToInt(buf.slice(64))
  // support both versions of `eth_sign` responses
  if (v < 27) {
    v += 27
  }
  
```
  (They subsequently remove the 27 later on)
  
  Open to either!